ghc.nix

```diff
@@ -189,6 +191,13 @@ hspkgs.shellFor rec {
     "--with-libdw-includes=${elfutils.dev}/include"
     "--with-libdw-libraries=${elfutils.out}/lib"
     "--enable-dwarf-unwind"
+  ] ++ lib.optionals withSystemLibffi [
+    "--with-system-libffi"
+    "--with-ffi-includes=${libffi.dev}/include"
+    "--with-ffi-libraries=${libffi.out}/lib"
+  ] ++ lib.optionals (crossTarget != null) [
```

Comment:
Hey wait a bit, this is wrong

Reply:
This should be `--host=${system}`, and even so, all configure options defined here are overwritten by env hook in `wasi-sdk` on purpose, there's really no need to fiddle with options here

Reply:
Is it?